flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/dialect/JDBCDialect.java

```diff
@@ -146,6 +153,42 @@ default String getSelectFromStatement(String tableName, String[] selectFields, S
 				.map(f -> quoteIdentifier(f) + "=?")
 				.collect(Collectors.joining(" AND "));
 		return "SELECT " + selectExpressions + " FROM " +
-				quoteIdentifier(tableName) + (conditionFields.length > 0 ? " WHERE " + fieldExpressions : "");
+			quoteIdentifier(tableName) + (conditionFields.length > 0 ? " WHERE " + fieldExpressions : "");
+	}
+
+	/**
+	 * Get create table statement.
+	 */
+	default String getCreateTableStatement(String tableName, TableSchema schema, String[] primaryKeyFields) {
```

Comment:
The same to `getDialectTypeName`. My concern is that, we should throw unsupported exception if the dialect doesn't implement `autoCreateTable`, rather than an unreadable exception.